docs/requester_pays.rst

```diff
@@ -0,0 +1,195 @@
+charges due to someone maliciously or accidentally downloading a large volume of data,
+we've set our cloud storage to use `requester pays
+<https://cloud.google.com/storage/docs/requester-pays>`__.
+
+"Requester pays" means the person downloading the data is responsible for those costs
+instead. As a user paying the data egress charges associated with your data usage is
+cheap, but as a data provider covering the data egress charges for all of your users can
+quickly get expensive! Downloading all of the EPA CEMS, FERC 1, PUDL, and US Census data
+we're publishing from North America should cost about $0.75, but if we had 1000
+downloads in a month that would cost of $750. The PUDL Intake catalog is also set up to
+try and cache the data locally so that it's not downloaded again until a new version is
+released.
+
+Setting up a GCP project for billing
+------------------------------------
+
+The following instructions assume you don't currently have GCP set up, and just want to
+be able to use public data that uses requester pays. If your organization already uses
+GCP or you want to access the PUDL Catalog in the context of another project that's
+running on GCP, you'll probably need to do something different.
+
+Create a GCP Account
+~~~~~~~~~~~~~~~~~~~~
+
+Go to `<https://cloud.google.com>`__ and click on the "Get Started for Free" button.  A
+prompt should appear asking you to choose which Google account to use for your
+GCP-related activities. You should be able to log in with a Gmail account or another
+Google ID. You don't need to use Gmail for your email. Your Google ID is just used for
+identification and authentication.
+
+Create a New GCP Project
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+GCP allows you to organize services, access, and billing under various Projects, which
+you name and define. In the context of accessing public requester pays data, the only
+thing it needs to do is identify who you are, and how you'd like to pay for any charges
+associated with the accessing the data. These charges are from Google. Catalyst doesn't
+charge a markup or get any money from this, it's just a way to help cover the costs of
+distributing the data.
+
+Let's create a project for use in accessing public data.  Here's the
+`GCP documentation on creating a new project
+<https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project>`__
+
+After you've logged in, at the top of the main GCP home page you should see a search
+field. Type in "Manage Resources" and at the top of the suggested searches that pop up
+you should see a "Products and Pages" section, which contains "Manage Resources". Click
+on that section, and you should come to the Manage Resources page.
+
+Click on the "Create Project" link near the top of the page:
+
+.. image:: images/01-manage-resources.png
+
+The only field you need to have filled in here is the Project Name. You can use the
```

Comment:
For me, Billing Account and Location are required fields as well as Project Name.